testdata/sqllogictests/xlsx.slt

```diff
@@ -57,3 +57,21 @@ select "HEADING" from read_excel('file://${PWD}/testdata/xlsx/multiple_sheets.xl
 # negatives are not allowed for infer_rows
 statement error
 select * from read_excel('file://${PWD}/testdata/xlsx/multiple_sheets.xlsx', sheet_name => 'other', infer_rows => -1);
+
+# create external table
+
+statement ok
+create schema 'xlsx';
+
+statement ok
+set search_path = 'xlsx';
```

Comment:
is there a reason to do this in a different schema?

Reply:
not really